subsys/bluetooth/host/Kconfig

```diff
@@ -59,16 +61,6 @@ config BT_HCI_TX_STACK_SIZE
 config BT_HCI_TX_STACK_SIZE_WITH_PROMPT
 	bool "Override HCI Tx thread stack size"
 
-config BT_HCI_ECC_STACK_SIZE
```

Comment:
Since we are removing a Kconfig that has existed for 3 years, I guess we should, at the very minimum, mention this in the release notes. 

Not sure if we need to raise and RFC for Kconfigs.

Reply:
Added release note. I am not sure if there is a reason why we should keep a Kconfig that configures non-existing thread.